aws/cloudformation/moz-single.json

```diff
@@ -487,7 +487,37 @@
           }
         ]
       }
+    },
+
+    "FxaCloudfrontPolicy" : {
+      "Type" : "AWS::CloudFront::Distribution",
+      "Properties" : {
+          "DistributionConfig" : {
+              "DefaultCacheBehavior" : {
+                  "AllowedMethods" : [ "GET", "HEAD", "OPTIONS" ],
+                  "TargetOriginId" : "FxaCloudfrontCDN",
+                  "ForwardedValues" : {
+                      "QueryString" : "false",
+                      "Cookies" : { "Forward" : "none" }
+                  },
+                  "ViewerProtocolPolicy" : "allow-all"
+              },
+              "Enabled" : "true",
+              "Origins" : [ {
+                  "DomainName" : { "Fn::Join" : [ "", [{"Ref" : "Subdomain"}, ".", {"Ref" : "HostedZone"}, "."]]},
```

Comment:
@vladikoff - that's very possible. I was a bad person and copy/pasted the line from above w/o understanding what it does.